<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-09-22T23:54:43 EDT" name="Default suite" finished-at="2022-09-22T23:55:32 EDT" duration-ms="49234">
    <groups>
    </groups>
    <test started-at="2022-09-22T23:54:43 EDT" name="Default test" finished-at="2022-09-22T23:55:32 EDT" duration-ms="49234">
      <class name="test1.LoginTest1">
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:test1.LoginTest1@71812481]" started-at="2022-09-22T23:54:43 EDT" name="beforeMethod" finished-at="2022-09-22T23:54:46 EDT" duration-ms="3738" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="LoginWithSpecialCharTest()[pri:1, instance:test1.LoginTest1@71812481]" started-at="2022-09-22T23:54:46 EDT" name="LoginWithSpecialCharTest" finished-at="2022-09-22T23:54:55 EDT" duration-ms="8504" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginWithSpecialCharTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:test1.LoginTest1@71812481]" started-at="2022-09-22T23:54:55 EDT" name="afterMethod" finished-at="2022-09-22T23:54:56 EDT" duration-ms="831" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:test1.LoginTest1@71812481]" started-at="2022-09-22T23:54:56 EDT" name="beforeMethod" finished-at="2022-09-22T23:54:59 EDT" duration-ms="3311" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="LoginWithEmptyEmail()[pri:2, instance:test1.LoginTest1@71812481]" started-at="2022-09-22T23:54:59 EDT" name="LoginWithEmptyEmail" finished-at="2022-09-22T23:55:07 EDT" duration-ms="8355" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginWithEmptyEmail -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:test1.LoginTest1@71812481]" started-at="2022-09-22T23:55:07 EDT" name="afterMethod" finished-at="2022-09-22T23:55:08 EDT" duration-ms="606" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:test1.LoginTest1@71812481]" started-at="2022-09-22T23:55:08 EDT" name="beforeMethod" finished-at="2022-09-22T23:55:11 EDT" duration-ms="2911" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="LoginWithEmptyPassword()[pri:3, instance:test1.LoginTest1@71812481]" started-at="2022-09-22T23:55:11 EDT" name="LoginWithEmptyPassword" finished-at="2022-09-22T23:55:19 EDT" duration-ms="8372" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginWithEmptyPassword -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:test1.LoginTest1@71812481]" started-at="2022-09-22T23:55:19 EDT" name="afterMethod" finished-at="2022-09-22T23:55:20 EDT" duration-ms="601" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:test1.LoginTest1@71812481]" started-at="2022-09-22T23:55:20 EDT" name="beforeMethod" finished-at="2022-09-22T23:55:23 EDT" duration-ms="2914" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="LoginWithwrongemailpasswordTest()[pri:4, instance:test1.LoginTest1@71812481]" started-at="2022-09-22T23:55:23 EDT" name="LoginWithwrongemailpasswordTest" finished-at="2022-09-22T23:55:31 EDT" duration-ms="8459" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Sorry, your login entry doesn't match our records. Please try again.] but found [This should be working, but something's gone wrong. Please try again soon. [500.GENERIC]]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Sorry, your login entry doesn't match our records. Please try again.] but found [This should be working, but something's gone wrong. Please try again soon. [500.GENERIC]]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at test1.LoginTest1.LoginWithwrongemailpasswordTest(LoginTest1.java:74)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginWithwrongemailpasswordTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:test1.LoginTest1@71812481]" started-at="2022-09-22T23:55:31 EDT" name="afterMethod" finished-at="2022-09-22T23:55:32 EDT" duration-ms="602" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- test1.LoginTest1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
